wrappers/Python/pytest/test_superancillary.py

```
@@ -0,0 +1,121 @@
+    with pytest.raises(ValueError):
+        sa.eval_sat(T, "", 1)
+    # Mismatched array lengths are rejected
+    with pytest.raises(ValueError):
+        sa.eval_sat_many(np.array([T, T]), "P", 1, np.zeros(1))
+
+
+def test_chebyshev_expansion_eval_and_invert():
+    from numpy.polynomial import chebyshev as C
+
+    xmin, xmax = 1.0, 3.0
+    f = lambda x: x ** 3 + 2.0 * x  # monotonic increasing on [1, 3]
```

Ruff (0.15.15)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `f` as a `def`

(E731)